src/lib/phase.ml

```diff
@@ -64,6 +65,10 @@ let conv =
         judge_artefact_descr; head_judge_artefact_descr;
       })
 
+
+let list st =
+  State.query_list ~st ~conv {|SELECT * FROM phases|}
```

Comment:
I don't see the point of listing all phases independant of competitions, as in, apart from testing on small databases, I don't see the frontend/backend/lib ever needing it.